tests/test_scalebar.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import pytest
+from morphocut import Node, Pipeline
+from morphocut.stream import Unpack
+from morphocut.scalebar import DrawScalebar, draw_scalebar
+
+
+@pytest.fixture
+def test_image():
+    return np.zeros((100, 100), dtype=np.uint8)  # Grayscale image
+
+
+@pytest.mark.slow
+def test_draw_scalebar_function(test_image):
+    length_unit = 100
+    result = draw_scalebar(length_unit)
+
+    # Assertions to verify the output
+    assert isinstance(result, np.ndarray)
+    assert result.shape[0] == 32
+    assert result.shape[1] == length_unit + 20  # 10 margin on each side
```

Comment:
This is only true if unit is "px" (`px_per_unit=1`). Could you maybe try different `px_per_unit` and check the expected result?